src/test/java/seedu/tasklist/logic/parser/ListCommandParserTest.java

```diff
@@ -20,16 +19,12 @@ public void parse_validArgs_returnsListCommand() {
 
     @Test
     public void parse_invalidArgs_throwsParseException() {
-        assertParseFailure(parser, "-1", String.format(
-                MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, "-1", ListCommand.MESSAGE_INDEX_NOT_IN_RANGE);
 
-        assertParseFailure(parser, "0", String.format(
-                MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, "0", ListCommand.MESSAGE_INDEX_NOT_IN_RANGE);
 
-        assertParseFailure(parser, "a", String.format(
-                MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, "a", ListCommand.MESSAGE_INDEX_NOT_IN_RANGE);
 
-        assertParseFailure(parser, "%", String.format(
-                MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, "%", ListCommand.MESSAGE_INDEX_NOT_IN_RANGE);
```

Comment:
May be good to include test cases where the index lies outside of the stated range (e.g 51)

Reply:
I havent done the test cases proper :') had to change the initial test cases because the range is different now